package.json

```diff
@@ -13,7 +17,8 @@
       "webextensions"
     ],
     "ignores": [
-      "extension/vendor/**"
+      "extension/vendor/**",
+      "Gruntfile.js"
```

Comment:
Because it keeps showing errors for the env vars and the non-camelcase keys. I can't change these keys to camelcase (https://github.com/sindresorhus/refined-github/blob/307e65918897a26bb31e154511460a5715abf292/Gruntfile.js#L8).

Either we ignore the file or add a rule to ignore `camelcase` in the whole repo.